.github/workflows/ci.yml

```diff
@@ -0,0 +1,54 @@
+# linting check of radis-app
+name: 🌱 CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    name: 🧪 Test
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: 'yarn'
+      - name: Install Node.js dependencies
+        run: yarn install --frozen-lockfile
+      - name: Linting check
+        run: cd frontend && yarn lint
+      - name: Type check
+        run: cd frontend && yarn type-check
+      - name: Run tests
+        run: cd frontend && yarn test
+  build:
+    name: 📦 Build and Upload Artifact
+    runs-on: ubuntu-latest
+    if : ${{ success() }}
+    needs: [test]
+    steps :
+      - name: Check out Git repository
+        uses: actions/checkout@v3
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: 'yarn'
+      - name: Install Node.js dependencies
+        run: yarn install --frozen-lockfile
+      - name: Build 
+        run: yarn build
+      - name: Artifact upload
+        uses: actions/upload-artifact@v3
+        with:
+          name: Radis App
+          path: ./frontend/build
```

Comment:
I don't think we need both this and https://github.com/suzil/radis-app/blob/main/.github/workflows/main.yml#L12.

Also, there's no reason to upload an artifact to GitHub as we are uploading the artifact to S3 (indirectly through the AWS CDK).

Reply:
ok @suzil